.github/workflows/test.yml

```diff
@@ -57,6 +57,24 @@ jobs:
       - name: Running Harvest tests
         run: make linux/harvest-tests
 
+  unit-test-macos:
+    name: Macos unit tests
+    runs-on: macos-10.15
```

Comment:
macos-11 giving errors

Reply:
which ones? I only get to see 10.15 errors on the GHA record

Reply:
Some "unexpected errors" : https://github.com/actions/virtual-environments/issues/841
Same screen as in the link :( 

I saw in [the comments](https://github.com/actions/virtual-environments/issues/841#issuecomment-756639419) that wasn't happening in 10 and didn't have any since I changed it